libretro/libretro-core.c

```diff
@@ -35,6 +35,50 @@ static char buf[64][4096] = { 0 };
 long microSecCounter=0;
 int cpuloop=1;
 
+#ifdef __WIN32__
+int numlock_original_state=0;
+
+#ifdef __WIN32__
+#include <windows.h>

```

Comment:
Are you adding external dependencies here? Are we sure the buildbot has these installed? Were these functions X* used before anywhere else in this core or in any other Win32 core?

This needs checking I think to make sure the buildbot can compile this stuff.